packages/web/components/OrderMessages/index.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useAuth } from '../../hooks';
+import * as S from './styles';
+import { dateToString } from '../../utils/helper';
+import { InputField } from '../Form';
+
+const chatMock = [
+	{
+		id: 1,
+		date: '2020-01-01 12:53:24.000000',
+		messages: [
+			{
+				id: 1,
+				ownMessage: true,
+				content: 'Olá',
+				created_at: '2020-01-01 14:53:24.000000',
+			},
+			{
+				id: 2,
+				ownMessage: false,
+				content: 'Oi, tudo bem?',
+				created_at: '2020-01-01 15:53:24.000000',
+			},
+			{
+				id: 3,
+				ownMessage: true,
+				content: 'Tudo bem, e você?',
+				created_at: '2020-01-01 15:59:24.000000',
+			},
+		],
+	},
+	{
+		id: 2,
+		date: '2020-12-25 12:53:24.000000',
+		messages: [
+			{
+				id: 1,
+				ownMessage: false,
+				content: 'Esqueci de te responder, feliz natal!',
+				created_at: '2020-01-01 14:53:24.000000',
+			},
+			{
+				id: 2,
+				ownMessage: true,
+				content: 'Tchau!',
+				created_at: '2020-01-01 15:53:24.000000',
+			},
+			{
+				id: 3,
+				ownMessage: false,
+				content: 'Que falta de educação cara',
+				created_at: '2020-01-01 15:54:24.000000',
+			},
+			{
+				id: 4,
+				ownMessage: false,
+				content: 'Só vim te desejar boas festas',
+				created_at: '2020-01-01 15:55:24.000000',
+			},
+			{
+				id: 5,
+				ownMessage: true,
+				content: 'Aqui não é facebook',
+				created_at: '2020-01-01 15:59:24.000000',
+			},
+		],
+	},
+];
+
+const OrderMessages = ({ isBuyer, currentOrder, backToList }) => {
+	const { user } = useAuth();
+
+	return (
+		<S.Container>
+			<div>
+				<div>
+					<S.Button onClick={backToList}>Voltar</S.Button>
+				</div>
+				<S.MessagesWrapper>
+					{chatMock.map((chat) => (
```

Comment:
It will be interesting to show a information message (or image) if there are no chat messages to show.

Reply:
For sure, will do this when integrating 👍